bpf/tests/ipsec_encryption_on_egress.c

```diff
@@ -52,21 +52,15 @@ PKTGEN("tc", "ipsec_encryption_on_egress")
 int ipsec_encryption_on_egress_pktgen(struct __ctx_buff *ctx)
 {
 	struct pktgen builder;
-	struct ethhdr *l2;
 	struct iphdr *l3;
 
 	pktgen__init(&builder, ctx);
 
-	l2 = pktgen__push_ethhdr(&builder);
-	if (!l2)
-		return TEST_ERROR;
-	ethhdr__set_macs(l2, (__u8 *)mac_one, (__u8 *)mac_two);
-
+	l3 = pktgen__push_ipv4_packet(&builder, (__u8 *)mac_one, (__u8 *)mac_two,
+				      v4_pod_one, v4_pod_two);
 	l3 = pktgen__push_default_iphdr(&builder);
```

Comment:
We don't need this line anymore, do we?

Reply:
yep accidental left-over. Cleaned it up :+1: